README.md

```diff
@@ -30,11 +30,23 @@ direction of the mouse travel. The fluid fades slowly over time so as not to fil
 the container.
 
 ### MPI for CUDA Backend 
-The source code within provides two SYCL example programs; send_recv_buff.cpp, send_recv_usm.cpp, which are CUDA MPI aware. One example uses the SYCL Unified Share Memory (USM). The other uses the Buffer (buff) model. Each example uses the programing pattern Send-Receive. 
+The source code within provides two SYCL example programs; send_recv_buff.cpp, 
+send_recv_usm.cpp, which use CUDA-Aware MPI. One example uses the SYCL Unified 
```

Comment:
For me this sounds like only send_recv_buff.cpp and send_recv_usm.cpp use CUDA-Aware MPI and makes it sound like maybe scatter_reduce_gather uses standard host MPI pointers.

The main information that the README.md should have in some form is that:
- The intention of the send_recv_usm.cpp and send_recv_buff.cpp are to be minimal examples of how to use gpu-aware MPI in sycl using either usm or buffers. 
- The third example also uses cuda-aware MPI and USM (since this is much more commonly used than buffers) but is slightly more complex.

Reply:
Expanded the description to be more explanatory.
Done.